contracts/governance/FeeSharingCollector/FeeSharingCollector.sol

```diff
@@ -56,14 +56,14 @@ contract FeeSharingCollector is
     using SafeERC20 for IERC20;
 
     address constant ZERO_ADDRESS = address(0);
-    address public constant RBTC_DUMMY_ADDRESS_FOR_CHECKPOINT =
-        address(uint160(uint256(keccak256("RBTC_DUMMY_ADDRESS_FOR_CHECKPOINT"))));
+    address public constant NATIVE_TOKEN_DUMMY_ADDRESS_FOR_CHECKPOINT =
+        address(uint160(uint256(keccak256("NATIVE_TOKEN_DUMMY_ADDRESS_FOR_CHECKPOINT"))));
 
     /* Events */
 
-    /// @notice Deprecated event after the unification between wrbtc & rbtc
+    /// @notice Deprecated event after the unification between wrappedNativeToken & nativeToken
     // event FeeWithdrawn(address indexed sender, address indexed token, uint256 amount);
-    event FeeWithdrawnInRBTC(address indexed sender, uint256 amount);
+    event FeeWithdrawnInNativeToken(address indexed sender, uint256 amount);
```

Comment:
Regarding changing event and function names, care needs to be taken so that frontend/backend will still work.